src/cdk/utils/basic-authorizer-handler/index.ts

```diff
@@ -39,10 +39,29 @@ function createAllowPolicy(
   };
 }
 
+function getHeaderValue(
+  headers: Record<string, string> | undefined,
+  name: string
+): string | undefined {
+  if (!headers) {
```

Comment:
Instead of this `if`-Statement you could use a default paramter: `headers: Record<string, string> = {}`

Reply:
Good idea! Fixed.